src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -574,11 +597,18 @@ bool GetPeerNodeInfo(CommandHandler * commandHandler, const ConcreteCommandPath
                           CHIP_ERROR err = gOtaSender.PrepareForTransfer(fabricIndex, nodeId);
                           if (CHIP_NO_ERROR != err) {
                               LogErrorOnFailure(err);
-                              handler->AddStatus(cachedCommandPath, Protocols::InteractionModel::Status::Failure);
-                              handle.Release();
-                              return;
+                              if (err == CHIP_ERROR_BUSY) {
+                                  response.status = static_cast<OTAQueryStatus>(MTROTASoftwareUpdateProviderOTAQueryStatusBusy);
+                                  handler->AddResponse(cachedCommandPath, response);
+                                  handle.Release();
+                                  return;
+                              } else {
```

Comment:
I expect the else after return will fail our lints.

Reply:
hmm i ll fall through.